docs/index.md

```diff
@@ -39,13 +37,12 @@ Versions for back-end software including Django, Wagtail, Jinja, etc. are kept i
 https://github.com/cfpb/cfgov-refresh/tree/master/requirements
 
 - `base.txt`: shortcut for `django.txt` + `wagtail.txt` + `libraries.txt`
-- `deployment.txt`: requirements for deployment, includes `base.txt` and `postgres.txt` and a New Relic library which we don't install anywhere else.
+- `ci.txt`: specific requirements for the continuous integration environment. Should/could be moved to CI configuration files?
```

Comment:
Did you want this question in the docs?

Reply:
I moved this line from below, where it was still called `travis.txt`, and that question came along with it. Seems like a pertinent question that we still should resolve.

Reply:
Question is good; just seems out of place on the main page of our docs.
Not a blocker.